resources/assets/less/bem/not-official-footer.less

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+.not-official-footer {
```

Comment:
what does this even mean. Just call it `development-deploy-footer`

Reply:
I just copy this class name from the issue. Will change.